### PR TITLE
Standard Linting

### PR DESCRIPTION
**Pull request by carloscuesta**
References #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,11 @@
   "homepage": "https://github.com/behaviorbot/update-docs#readme",
   "devDependencies": {
     "eslint-config-probot": "^0.1.0",
```
Comment (hiimbex) [inline package.json]: nit: `eslint-config-probot` is no longer used and should be removed from the `package.json`

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "mocha && xo --extend probot"
+    "test": "mocha && standard --extend probot"
```
Comment (hiimbex) [inline package.json]: nit: `--extend probot` probot is no longer needed, it should be `mocha && standard`

File: index.js [hunk 3]
```diff
@@ -1,38 +1,35 @@
-/* eslint array-callback-return: 0 */
-/* eslint prefer-arrow-callback: 0 */
-
```
Comment (hiimbex) [inline index.js:3]: good catch removing these! 👍

**Review by hiimbex — CHANGES_REQUESTED** [review]
Looks great! 2 minor nits and then this should be good to go

**Comment by carloscuesta** [comment]
Oops! Fixed now 👌

**Review by hiimbex — APPROVED** [review]
Awesome, thanks! Happy hacktoberfest hacking! 🎃

**Pull request merged**